aea/configurations/base.py

```diff
@@ -205,6 +207,19 @@ class PackageConfiguration(Configuration, ABC):
     - contracts
     """
 
+    __slots__ = (
```

Comment:
why slots here?
I don't think it will save much memory, cause configuration instances amount is quite low during the agent life cycle in comparison to dialogues and messages

Reply:
Yes, but there is no cost to adding it and a small benefit.